suitesparse_graphblas/__init__.py

```diff
@@ -37,8 +37,173 @@ def initialize(*, blocking=False, memory_manager="numpy"):
     elif memory_manager == "c":
         lib.GrB_init(blocking)
     else:
-        raise ValueError(f'memory_manager argument must be "numpy" or "c"; got: {memory_manager!r}')
+        raise ValueError(
+            f'memory_manager argument must be "numpy" or "c"; got: ' "{memory_manager!r}"  # noqa
+        )
 
 
 __version__ = get_versions()["version"]
 del get_versions
+
+
+def libget(name):
+    """Helper to get items from GraphBLAS which might be GrB or GxB"""
+    try:
+        return getattr(lib, name)
+    except AttributeError:
+        ext_name = f"GxB_{name[4:]}"
+        try:
+            return getattr(lib, ext_name)
+        except AttributeError:
+            pass
+        raise
+
+
+bool_types = (lib.GrB_BOOL,)
+
+signed_integer_types = (
+    lib.GrB_INT8,
+    lib.GrB_INT16,
+    lib.GrB_INT32,
+    lib.GrB_INT64,
+)
```

Comment:
Why not `set` or `frozenset`?

Reply:
I only iterate them but yeah, a frozenset makes more sense for general lookup, I'll change it.